source/app/components/SearchTypeTabs/SearchTypeTabs.js

```diff
@@ -59,7 +59,7 @@ function SearchTypeTabs({ selectedSearch, setSelectedSearch }) {
         className={cs(styles.tab, selectedSearch === "es" && styles.selected)}
         onClick={selectES}
       >
-        Elasticsearch
+        Amazon ES
```

Comment:
Why not mention Amazon Elasticsearch completely here too?

Reply:
This comes up on the comparison tab and it may get a bit "large" on the tab name to git 3 tabs on the same page